reana/cli.py

```diff
@@ -1905,38 +1946,77 @@ def run_example(
 @click.option(
     "-m",
     "--mount",
+    "mounts",
+    multiple=True,
+    callback=volume_mounts_to_list,
+    help="Which local directories whould be mounted in the Kind nodes? local_path=reana_system_path",
+)
+@click.option(
+    "--debug",
     is_flag=True,
     default=False,
-    help="Should we mount /var/reana from host?"
+    help="Should we mount the REANA source code for live code updates?",
+)
+@click.option(
+    "-n", "--worker-nodes", default=0, help="How many worker nodes? default 0"
```

Comment:
Please change to `-w`, `--worker-nodes` (or for `-c`, `--count-worker-nodes`) so that `-n` will be reserved for `--namespace` everwhere.

Reply:
(or we just don't use short one-letter flag for this option?  Long is enough; the flag won't be used every day. In this way we don't have to invent too fantasy flag name just so that the leading letter would be free)